src/main/java/com/transferwise/openbanking/client/api/vrp/RestVrpClient.java

```diff
@@ -332,6 +331,20 @@ public OBDomesticVRPDetails getDomesticVrpDetails(String domesticVrpId, AspspDet
         return domesticVrpDetailsResponse;
     }
 
+    private OBDomesticVRPConsentResponse replaceRevokedStatusInDomesticVRPConsent(String jsonString) {
```

Comment:
Discussed in person to move to Deserlializer

Reply:
Yes, This is hacky and should not be done in normal scenario. 
NatWest will take their own time to resolve this issue, but meanwhile we should unblock the user to transfer money. 
It looks to me this error will persistent and completely block user from making any transaction.

Reply:
Once consent is Revoked a customer will be unable to initiate a VRP payment anyway. 
What I am saying that this should be considered as part of this ticket

Reply:
Added 
Fixed 'get consent failures when bank send Revoked status enum': OB VRP v3.1.9 specification doesn't specify Revoked state
  for 'OBDomesticVRPConsentResponseData.StatusEnum' but some ASPSP still sends it. We fixed it by adding
  VrpConsentResponseStatusEnumDeserializer which replace Revoked with Rejected while JSON deserializing
in [7a5751f](https://github.com/transferwise/openbanking-client/pull/180/commits/7a5751fbfdc46c88d1889807a5865b4b691270a4).

Reply:
Okay, I won't close the ticket until this is fixed from NatWest side.